blog/_layouts/home.html

```diff
@@ -0,0 +1,11 @@
+---
+layout: default
+---
+
+{% assign sections = content | split: "<!--faq -->" %}
```

Comment:
ouh, j'avais pas envisagé une idée comme ça !

Reply:
Et du coup ça te semble efficace comme ça ?

Reply:
y'a une idée qui a poppé dans ma tête en allant me coucher

Je me demande si on pourrait généraliser, genre, dans le contenu en markdown, on mettrait : 
```md
<!-- section : faq-->
abra

<!-- section : yolo-->
cadabra
```

Et dans le contenu rendu, on aurait:
```html
<section class="faq">
abra
</section>
<section class="yolo">
cadabra
</section>
```

Et côté CSS, la personne qui créé la page peut styler en se basant sur le nom de classe

La question que je n'ai pas creusé, c'est si c'est possible tout court. Genre, est-ce qu'on peut faire un split en liquid qui se base sur une regexp ? (et capturer un morceau pour le réutiliser dans le rendu ?)

Reply:
(et je n'ai pas répondu à ta question... parce que je n'ai pas encore d'avis formé. Ça marche et donc, ça passe dans un MVP)

Reply:
A creuser !

Reply:
J'ai regardé un peu. Les regex ne sont pas utilisées avec Liquid. Par contre on peut faire notre propre module.

Reply:
Yes
Une autre option (plus fainétante que faire notre propre module) : 
ptèt qu'on peut tenter de chercher `<!-- section -->`
et dans le CSS, on stylise avec les sélecteur [`:nth-of-type`](https://developer.mozilla.org/en-US/docs/Web/CSS/:nth-of-type) (genre `section:nth-of-type(3)`), comme ça, c'est un peu plus générique que chercher spécifiquement `<!--faq -->`